Dockerfile

```diff
@@ -37,7 +37,7 @@ USER 1001
 RUN /tmp/scripts/assemble
 
 # These manual pip installs will be removed before final release and added to the Piplock file
-RUN pip install openvino-dev
+RUN pip install openvino-dev tensorflow~=2.4.1 tensorflow-serving-api --use-deprecated=legacy-resolver
```

Comment:
Why did you move the tensorflow installs from the pipfile to here?

Reply:
pipenv was not able to do the dependency resolution. once we have 2021.4 this will be easier